scripts/format.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# format the source files
+if [ "$#" -eq 0 ]
+then
+  # format source files
+  yapf -i -r --style scripts/setup.cfg securedrop bin
```

Comment:
I think once you move `setup.cfg` to root, you no longer need to manually specify style (but i could be wrong). If this is true, then you can remove style option.

Reply:
Done. And you are correct.

Reply:
Done